tests/python_package_test/test_engine.py

```diff
@@ -1068,6 +1068,39 @@ def test_cvbooster():
     assert ret < 0.15
 
 
+def test_cvbooster_save_load():
+    X, y = load_breast_cancer(return_X_y=True)
+    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.1, random_state=42)
+    params = {
+        'objective': 'binary',
+        'metric': 'binary_logloss',
+        'verbose': -1,
+    }
+    nfold = 3
+    lgb_train = lgb.Dataset(X_train, y_train)
+
+    def predict(cv_booster):
+        return np.array(cv_booster.predict(X_test))
+
+    cv_res = lgb.cv(params, lgb_train,
+                    num_boost_round=25,
```

Comment:
```suggestion
                    num_boost_round=10,
```

We try to keep these tests as inexpensive as possible, to ensure that they run quickly. Could `num_boost_round` be reduced to 10?

Reply:
👍